### PR TITLE
[bitnami/external-dns] add parameters for OVH authentication

### PR DESCRIPTION
**Pull request by jgaillardet**
**Description of the change**

Add to the chart 3 new parameters for OVH authentication:
- ovh.consumerKey
- ovh.applicationKey
- ovh.applicationSecret

These parameters are defined as env vars in the [tutorial manifest](https://github.com/kubernetes-sigs/external-dns/blob/master/docs/tutorials/ovh.md#manifest-for-clusters-with-rbac-enabled).

**Benefits**

Users will be able to deploy the chart for OVH

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

None

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/external-dns/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: external-dns
-version: 2.20.11
```
Comment (miguelaeh) [inline bitnami/external-dns/Chart.yaml:3]: Please bump the minor version

Comment (jgaillardet) [reply bitnami/external-dns/Chart.yaml:3]: Hi, minor version bumped

**Comment by miguelaeh** [comment]
Hi @jgaillardet ,
You have bumped the patch version instead of the minor. The versions are `MAJOR.MINOR.PATCH`: https://semver.org/

**Comment by jgaillardet** [comment]
Hi @miguelaeh,
Chart version set to 2.21.0

**Pull request merged**